python_basics_lists_dictionaries/python_basics_lists_dictionaries.md

```diff
@@ -394,6 +409,14 @@ You might have noticed by now that we have used square brackets in a few differe
 
 </div>
 
+<div class = "care">
+<b style="color: rgb(var(--color-highlight));">A little encouragement...</b><br>
+
+We've just gone through quite a few useful list and dictionary methods, and you might be concerned about remembering them all! There's no need to memorize them all right now-- it's likely that you'll start to remember them as you use them more, but in the meantime, feel free to refer back to this module whenever you need to. 
```

Comment:
```suggestion
We've just gone through quite a few useful list and dictionary methods, and you might be concerned about remembering them all! There's no need to memorize them all right now-- it's likely that you'll start to remember them as you use them more, but in the meantime, refer back to this module whenever you need to. 
```
This is more about the way I personally feel about the phrase "feel free to" which I use a lot but also sounds like more like permission than an explicit invitation/instruction. Feel free (haha) to accept or ignore this suggestion.

Reply:
I see what you mean!